Mupa1/StackOverFlowLite1
# Ft all questions 162692447

__What does this PR do__
Adds all questions page for the user

__Description of task to be completed__
Create a descriptive sign up page with the following features:-
    1. All questions
    2. Links to other pages

__How should this be manually tested__
    1. Clone this repository
    2. Cd into the StackOverFlowLite1 folder
    3. Open all_questions.html in your browser

__What are the relevant pivotal tracker stories__
#162692447

__Screenshot__
![all_question_screenshot](https://user-images.githubusercontent.com/44979953/50078824-07272080-01f9-11e9-89dd-8ed31b64e27b.png)
 A User should be able to see all questions
